experimental/dds/tree2/src/test/shared-tree/sharedTree.spec.ts

```diff
@@ -1968,37 +1963,6 @@ describe("SharedTree", () => {
 	});
 });
 
-/**
- * Updates the given `tree` to the given `schema` and inserts `state` as its root.
- */
-// TODO: replace use of this with initialize or schematize, and/or move them out of this file and use viewWithContent
-function initializeTestTree(
```

Comment:
I'd avoid proliferating this any further: just inline what you need into the test trees where you use it.